demosplan/DemosPlanMapBundle/Utilities/MapScreenshotter.php

```diff
@@ -580,18 +570,6 @@ protected function adjustPictureSize(float $top, stdClass $viewport, float $bott
         }
     }
 
-    /**
-     * @param resource $resource
-     */
-    public function assertResource($resource): bool
-    {
-        if (false === is_resource($resource)) {
-            throw new InvalidArgumentException(sprintf('Argument must be a valid resource type. %s given.', gettype($resource)));
-        }
-
-        return true;
-    }
-
```

Comment:
Still used in unit tests.

Reply:
Adjusted in [51892aa](https://github.com/demos-europe/demosplan-core/pull/637/commits/51892aaf334ff1eee681ceb591bc9342ff3b3bdb)